configserv/image.yaml

```diff
@@ -1,4 +1,4 @@
-version: 1.0
+version: latest
```

Comment:
> The latest tag you get always in the build system (as well, as others). 

I'm not sure i understand this part. Which build system are you referring to?

My impression was that this is the version used for the docker image that gets built. We are using the 'latest' docker tag as a way to pull down the latest docker image of each component when doing system testing.